README.md

```diff
@@ -20,114 +20,8 @@ code for Delivery Optimization Agent on Linux.
-
-```markdown
-> cd <project root>
-```
-
-We provide an easy-to-use python script for building our client components from the project root, you can inspect build.py for additional build flags  
-On debian-based systems, run this command to build the client and package it as a .deb file
-
-```markdown
-> python3 build/build.py --project agent --package-for deb
-```
-
-Run this command to build the sdk
-
-```markdown
-> python3 build/build.py --project sdk --package-for deb
-```
-
-In order to build the plugin, you must build & install the sdk, an easy way to do this is to install the the packages you produced in the previous two steps
-
-Navigate to the build output directory for the agent and install the agent package
-
-```markdown
-> cd /tmp/build-deliveryoptimization-agent/linux-debug/
-> sudo apt-get install ./deliveryoptimization-agent*.deb
-```
-
-The sdk produces a runtime and development package, in this case you'll want to install both  
-Navigate to build output directory for the sdk and install both packages
-
-```markdown
-> cd /tmp/build-deliveryoptimization-sdk/linux-debug/
-> sudo apt-get install ./libdeliveryoptimization*.deb
-```
-
-With the sdk installed, you can now build the plugin by navigating back to the project root
-
-```markdown
-> cd <project root>
-> python3 build/build.py --project plugin-apt --package-for deb
-```
-
-At this point, you should have built and packaged all components
-
-
-### Testing DO Client components
-
-As guidance, please ensure proper code coverage for project contributions  
-Unit tests for the agent and sdk are produced as a part of the above build command, you can find them in the build output directory
-
-```markdown
-> cd /tmp/build-deliveryoptimization-agent/linux-debug/client-lite/test
-```
-
-Our tests utilize the [GTest](https://github.com/google/googletest) unit testing framework, which supports test filtering via command line  
-You can run all agent tests by running
-
-```markdown
-> ./deliveryoptimization-agent-tests
-```
-
-You can filter for specific tests as well, reference the GTest documentation for filtering rules and syntax
-```markdown
-> sudo ./deliveryoptimization-agent-tests --gtest_filter=DownloadManagerTests*
-```
-
-The test executable for the SDK is located in the sdk build output as well
-
-```markdown
-> cd /tmp/build-deliveryoptimization-sdk/linux-debug/sdk-cpp/tests
-```
-
-The sdk tests expect a running do-agent, you can either manually run the agent executable from its build output or install the agent package as you may have done while building the plugin  
-You can run the sdk tests just like the agent tests
-
-```markdown
-> sudo ./deliveryoptimization-sdk-tests
-```
-
-And filter them similarly
-
-```markdown
-> sudo ./deliveryoptimization-sdk-tests --gtest_filter=DownloadTests*
-```
+## Build and Test
+* [Getting Started on Linux](https://github.com/microsoft/do-client/tree/main/docs/getting-started-on-linux)
```

Comment:
>https://github.com/microsoft/do-client/tree/main/docs/getting-started-on-linux [](start = 29, length = 78)

I think markdown supports relative links. (docs/getting-started-on-linux.md) should work.

Reply:
Prefer using the full path if this readme ever changes. Either way this is a formatted link so to an enduser they only see [Getting Started on Linux]

---
In reply to: [605244606](https://github.com/microsoft/do-client/pull/17#discussion_r605244606) [](ancestors = 605244606)

Reply:
I suggested relative link to avoid specifying the repo name, branch name but sure this isn't critical. By the way, don't you need the .md suffix in the link?

---
In reply to: [605253187](https://github.com/microsoft/do-client/pull/17#discussion_r605253187) [](ancestors = 605253187,605244606)

Reply:
I referenced the adu readme and they don't have the .md extension.

See: 
[https://raw.githubusercontent.com/Azure/iot-hub-device-update/main/README.md](https://raw.githubusercontent.com/Azure/iot-hub-device-update/main/README.md)

---
In reply to: [605284599](https://github.com/microsoft/do-client/pull/17#discussion_r605284599) [](ancestors = 605284599,605253187,605244606)